### PR TITLE
Add meeting notes from 2023-01 (January 2023)

### PR DESCRIPTION
**Pull request by CAM-Gerlach**
Since I didn't see anyone else adding them, I figured I'd go ahead and do so. The meeting notes from January, removing the leftover bullet points we didn't get the chance to discuss, and with some light formatting cleanup (as a separate commit, for full transparency). I wasn't sure if I should add links to Petr's PR with the SVGs or updates on anything else, since that was well after the meeting.

### CODE REVIEW

**Comment by CAM-Gerlach** [comment]
Maybe worth implementing python/core-workflow#460 here for triagers and core devs? I can't tag @hugovk for review, which I suspect may be because he doesn't even have the Write or even Triage role here, despite being a core dev, PEP editor, docs team/proofreader member, etc.,

File: docs/monthly-meeting/2023-01.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+- (Hugo) SEO improvements. Using Sphinx extension that generate images for every single page that we have. But it is too heavy for CPython docs (40 MBs and nearly 500 images), using Matplotlib to do it, which is pretty heavy library. Contributors shouldn't need to be installing this additional library, so we need to find solutions. The extra SEO images should only be generated when publishing CPython docs, and not for building locally. There should be an option to do it.
+  - Check with Ee re. the file sizes
+
+- (CAM) PR for PEPs PR templates submitted: [python/peps#2956](https://github.com/python/peps/pull/2956)
+
+- (CAM) What's New issues.
+  * [python/cpython#100734](https://github.com/python/cpython/issues/100734)
+    Syncing changelog/What's New between versions
+
+    * CAM will sync things manually for now, we might waant some automation in the future
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2023-01.md:38]:
```suggestion
    * CAM will sync things manually for now, we might want some automation in the future
```

File: docs/monthly-meeting/2023-01.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+- Time for the meeting
+  - Let's alternate the Monday slot with another time, so other people can join?
+  - This is probably not the best audience :)
+
+
+### Python Project Documentation
+
+- [Trouble installing Python](https://mail.python.org/archives/list/docs@python.org/thread/I7JDNUYWIZ3QVY33IDYWFKDTZMIPIVNS/)
+  - [Running scripts with a GUI](https://docs.python.org/3/using/mac.html#running-scripts-with-a-gui)
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2023-01.md:63]:
```suggestion
    - [Running scripts with a GUI](https://docs.python.org/3/using/mac.html#running-scripts-with-a-gui)
```

**Review by ezio-melotti — APPROVED** [review]

**Comment by encukou** [comment]
Thanks for getting to this!

**Pull request merged**